src/styles/design-system.css

```
@@ -47,6 +48,7 @@
 
   .ui-site-nav a {
     @apply text-white no-underline transition-colors;
+    padding-block: 12px;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)